package.json

```diff
@@ -77,7 +77,7 @@
     "react-transition-group": "^2.7.0",
     "react-virtualized": "^9.18.5",
     "react-virtualized-select": "^3.1.3",
-    "recharts": "^1.0.0-beta.10",
+    "recharts": "1.7",
```

Comment:
Why 1.7? Current version is 1.8.5
https://github.com/recharts/recharts/blob/master/package.json

Reply:
logarithmic scale is broken for bars